### DISCUSSION TITLE
How do I enable sticky footer in this theme?

### DISCUSSION THREAD

**Discussion by kun-codes** [Speyll/anemone#8] [Q&A] [answered]
I want something like this in the theme: https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Sticky_footers

where no matter how short is the content, the footer always stays at the bottom of the page instead of just under the main content

**Comment by kun-codes** [comment]
@Speyll can you help?

**Comment by Speyll** [comment, marked as answer]
Implemented in the last commit 👍
